printer_mods/mastur_mynd/Stealth_Zero/CAD/README.md

```diff
@@ -0,0 +1 @@
+This project is in active development. This CAD is only made available for reference or personal modifications. I do not make any guarantees that objects in the CAD are tested or functional. For functional parts please refer to the printable STL files in the other directories.
```

Comment:
The CAD Files you submit *need* to match the stl files you provide.   I realize this notice is probably for use in your own repository, but it can't be here.